incuna-sass/components/_forms.sass

```diff
@@ -1,310 +1,84 @@
 $help-text-font-size: 80% !default
+$form-error-color: red !default
 
 form
-    margin: $form-margin
+    max-width: $form-max-width
 
 fieldset
     padding: 0
     margin: 0
     border: 0
 
 .control-group
+    width: 100%
     margin:
-        bottom: $field-margin
+        bottom: 1em
 
 // Groups of fields with labels on top (legends)
 legend
     display: block
-    width: 100%
-    padding: 0
-    margin-bottom: $field-margin
-    line-height: $field-margin
-    border: 0
-    border-bottom: 1px solid
-
-// Identify controls by their labels
-label
-    display: block
-    margin:
-        bottom: $label-margin
-    &[for]
-        cursor: pointer
-
-// Inputs, Textareas, Selects
-@mixin field-selectors
-    input,
-    textarea,
-    select
-        @content
-
-@mixin button-inputs
-    button,
-    input[type="submit"],
-    input[type="reset"],
-    input[type="button"]
-        @content
-
-@mixin text-inputs
-    input
-        // https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Input
-        &[type="color"],
-        &[type="date"],
-        &[type="datetime"],
-        &[type="datetime-local"],
-        &[type="email"],
-        &[type="month"],
-        &[type="number"],
-        &[type="password"],
-        &[type="search"],
-        &[type="tel"],
-        &[type="text"],
-        &[type="time"],
-        &[type="url"],
-        &[type="week"]
-            @content
-
-@include field-selectors
-    display: inline-block
-    vertical-align: middle
-    width: $field-width
-    height: $field-height
-    padding: $field-padding
-    font:
-        size: $field-font-size
-    border: 1px solid
-    &:focus
-        outline: 0
-    // Inputs within a label
-    label &
-        display: block
-
-@include text-inputs
-    // Reset iOS text inputs
-    -webkit-appearance: none
-    -webkit-border-radius: 0
-
-// Mini reset for unique input types
-input[type="image"],
-input[type="checkbox"],
-input[type="radio"]
-    width: auto
-    height: auto
-    padding: 0
-    margin: rem(3px) 0
-    border: 0
-    line-height: normal
-    cursor: pointer
-    .lt-ie8 &
-        margin:
-            top: 0
-
-// Reset the file input to browser defaults
-input[type="file"]
-    width: auto
-    padding: initial
-    line-height: initial
-    border: initial
-    background-color: initial
-
-// Help out input buttons
-input[type="button"],
-input[type="reset"],
-input[type="submit"]
-    width: auto
-    height: auto
-
-// Set the height of select and file controls to match text inputs
-select,
-input[type="file"]
-    // In IE7, the height of the select element cannot be changed by height, only font-size
-    height: rem(28px)
-    line-height: rem(28px)
-    .lt-ie8 &
-        //  For IE7, add top margin to align select with labels
-        margin:
-            top: rem(4px)
-
-// Reset line-height for IE
-input[type="file"]
-    .lt-ie10 &
-        line-height: rem(18px)
-
-// Chrome on Linux and Mobile Safari need background-color
```

Comment:
seems like it would depend on circumstances, it's not a very helpful comment really, can't see anything obviously about this. Possibly it's to remove the gradient background which is not necessarily something we always want to do.

I'm uncomfortable keeping code in here that I don't understand the purpose of because we _might_ need it. If I can figure out a reason for it that's fine, but I can't with this one